modules/rosa-planning-considerations.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// rosa_planning/rosa-planning-environment.adoc
+
+[id="initial-planning-considerations_{context}"]
+= Initial planning considerations
+
+Consider the following tested object maximums when you plan your {product-title} cluster.
+
+These guidelines are based on a cluster of 102 workers in a multi-availability zone configuration. For smaller clusters, the maximums are lower.
+
+The sizing of the control plane and infrastructure nodes is dynamically calculated during the installation process, based on the number of worker nodes. If the number of workers is changed after the installation, control plane and infra nodes will not be resized.
```

Comment:
@jonquilwilliams The control plane and infra nodes should be resized based on the compute node even after the installation. See https://issues.redhat.com/browse/SDA-2102 and https://issues.redhat.com/browse/SDA-3907